CHANGELOG.md

```diff
@@ -1,3 +1,697 @@
+# 1.4.0 (2022-08-26)
```

Comment:
Did Micah want to add something here?

Reply:
We'll do an update later with the additions Micah wants to make. The commit list is correct as-is, so we'll get this merged now.